infra/ansible/playbooks/aggregator.yaml

```diff
@@ -0,0 +1,120 @@
+- name: Run setup playbook
+  ansible.builtin.import_playbook: setup.yaml
+  vars:
+    host: aggregator
+
+- name: Run go playbook
+  ansible.builtin.import_playbook: go.yaml
+  vars:
+    host: aggregator
+
+- name: Run eigenlayer-cli playbook
```

Comment:
Eigenlayer cli is not needed for the aggregator

Reply:
Got it, it's removed now.